package.json

```diff
@@ -261,10 +260,10 @@
     "typescript": "3.7.5"
   },
   "dependencies": {
-    "metals-languageclient": "0.1.14",
+    "coc.nvim": "0.0.74",
+    "metals-languageclient": "0.1.15",
     "promisify-child-process": "^3.1.3",
-    "semver": "^7.1.2",
```

Comment:
I've removed `semver` since it's now unused by `coc-metals` (its usage has been moved to `metals-languageclient`)